components/governance/org.wso2.carbon.governance.rest.api/src/main/java/org/wso2/carbon/governance/rest/api/Asset.java

```diff
@@ -449,10 +457,26 @@ private Response createGovernanceAsset(String assetType, DetachedGenericArtifact
         String shortName = Util.getShortName(assetType);
         if (validateAssetType(shortName)){
 
-            if(isContentType(shortName , genericArtifact)){
-                //TODO
-                Response.status(Response.Status.CONFLICT).build();
-                //createContentAsset(shortName, genericArtifact);
+            if (isContentType(shortName, genericArtifact)) {
+                String[][] propertyArray = new String[2][2];
+                propertyArray[0][0] = "version";
+                propertyArray[0][1] = genericArtifact.getAttribute(OVERVIEW_VERSION);
+                propertyArray[1][0] = "resource.source";
+                propertyArray[1][1] = "GovernanceAPI";
+
+                Registry registry = getUserRegistry();
+
+                // overview_path is not a mandatory for known asset types.
+                // for unknown asset types '/_system/governance/' will be appended by the system
+                boolean isSuccessful = importResourceWithRegistry(registry,
+                        genericArtifact.getAttribute("overview_path") + genericArtifact.getAttribute(OVERVIEW_VERSION),
```

Comment:
Can you please explain about this overview_path attribute?label:tome  (Comment is not very clear)
What are the values coming for it?
Is this a common attribute to rxts?